### PR TITLE
[Chore]: hosting on heroku

### PR DESCRIPTION
**Pull request by sojida**
__What does this PR do?__
- This shows the set up to hosting on heroku

__Description of Task to be completed?__
- Host app on Heroku

__How should this be manually tested?__
- [Click here](https://stark-brook-15864.herokuapp.com/)

__Screenshots__
N/A

### CODE REVIEW

**Comment by amaechi-chuks** [comment]
> * This shows the

Please, can you clarify this?

**Review by amaechi-chuks — APPROVED** [review]

**Review by amaechi-chuks — COMMENTED** [review]
Good job, go ahead and merge

**Comment by sojida** [comment]
> > * This shows the
> 
> Please, can you clarify this?

Done. Thanks for point that out.

File: procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: npm start
```
Comment (vic3king) [inline procfile]: please fix EOL.

**Review by vic3king — APPROVED** [review]

File: server/src/models/index.js [hunk 2]
```diff
@@ -8,15 +8,14 @@ const env = process.env.NODE_ENV;
...
-// }
+let sequelize;
+if (config.url) {
+  sequelize = new Sequelize(process.env.DATABASE_URL, config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
 
-const sequelize = new Sequelize(config.database, config.username, config.password, config);
+// const sequelize = new Sequelize(config.database, config.username, config.password, config);
```
Comment (Chiazokam) [inline server/src/models/index.js]: Codes commented out should be removed totally

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "An app that actually takes note...lol",
   "main": "index.js",
   "scripts": {
-    "start": "npm run build:start",
+    "start": "node ./server/build/index.js",
```
Comment (Chiazokam) [inline package.json:7]: Any particular reason why you changed the script?

**Review by Chiazokam — CHANGES_REQUESTED** [review]

File: server/src/models/index.js (hunk 2 above)
Comment (sojida) [reply server/src/models/index.js]: I'll remove it. Thanks.

**Review by Chiazokam — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Chiazokam) [reply package.json:7]: No response yet?

Comment (sojida) [reply package.json:7]: I realized that Heroku runs the build script eventually.

Comment (Chiazokam) [reply package.json:7]: Ohh owkk.

**Pull request merged**